composer.lock

```diff
@@ -23172,8 +23172,6 @@
         "drupal/openid_connect": 10,
         "drupal/password_policy": 10,
         "drupal/regex_redirect": 15,
-        "drupal/search_api": 20,
```

Comment:
Not sure why this is being removed. Are the stability flags not necessary for all cases?